h/static/scripts/annotator/adder.js

```diff
@@ -221,6 +229,13 @@ function Adder(container, options) {
       'annotator-adder--arrow-up': arrowDirection === ARROW_POINTING_UP,
     });
 
+    // Some sites make big assumptions about interactive
+    // elements on the page. Some want to hide interactive elements
+    // after use. So we need to make sure the button stays displayed
+    // See: https://github.com/hypothesis/client/issues/137
+    this.element.querySelector(ANNOTATE_BTN_SELECTOR).style.display = 'block';
+    this.element.querySelector(HIGHLIGHT_BTN_SELECTOR).style.display = 'block';
```

Comment:
Does the page set this property via an inline style or a class? If they set it via an inline style, we could presumably achieve the same effect via `style.display = ''`, and that way we'll restore the display property originally set in the CSS.